keyboards/hineybush/hineyg80/readme.md

```diff
@@ -0,0 +1,17 @@
+# hineyg80
+
+hineyg80
```

Comment:
This line should be deleted unless you're going to include a photo of the PCB/keyboard here.

```suggestion
```

or 

`![hineyg80](a_direct_image_url_here)`